lib/src/main/java/co/nimblehq/alpine/lib/model/Biometrics.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.alpine.lib.model
+
+import android.graphics.Bitmap
+
+data class Biometrics(
+    val faceImageBitmap: Bitmap?,
+    val faceImageBase64: String?,
+    val portraitImageBitmap: Bitmap?,
+    val portraitImageBase64: String?,
+    val signatureBitmap: Bitmap?,
+    val signatureBase64: String?,
```

Comment:
Since we have the `Image` class, shall we use it here?
```suggestion
    val faceImage: Image?,
    val portraitImage: Image?,
    val signatureImage: Image?,
```